### ISSUE TITLE
Add support for .cue/.bin disk images

### ISSUE THREAD

**Issue by mihaip** [mihaip/infinite-mac#205]
e.g. for https://archive.org/details/PanicDiscPANIC.COM

**Comment by that-ben** [comment]
Does not seem to work with that .bin file: https://www.macintoshrepository.org/3226-dark-colony

![dasdsadas](https://github.com/mihaip/infinite-mac/assets/13210399/44d4bc44-4783-44f9-b335-69c3dddc90fe)

**Comment by mihaip** [comment]
I added support for .bin files imported as CD-ROM URLs only. Do you think people will also expect to use them dragged in files?

**Comment by that-ben** [comment]
I honestly don't know.  I consider .bin/.cue (and .bin/.toc) files to be pretty rare to begin with and it's not like it's hard to convert the .bin from a .bin/.cue file set to a .iso using ISO programs such as PowerISO anyway (literally 2 mouse clicks and less than 5 seconds wait for a 600MB CD image).

**Comment by Shogun38** [comment]
Hello - I came across this project ripping some Mac related CDs from MacAddict and MacHome (bin/cue). The MacHome CD in particular I have no idea whats on it. I am uploading it to the Internet Archive and was looking for an easy way to just spin up the disc and see whats on it to provide a description for the upload. I saw your software can load via URL for now so I'll just upload it and then try to load it via the URL. I just wanted to provide a use case for drag and dropping a bin file. This is a cool project!
